contrib/ansible/dashboard/inventory

```diff
@@ -0,0 +1,22 @@
+[servers:children]
+ui-server
+gurbir-server
+
+[servers:vars]
+pbench_url="http://{{ inventory_hostname }}"
+run_index="run."
+
+[ui-server]
+dhcp31-96.perf.lab.eng.bos.redhat.com
```

Comment:
We should not check in this inventory file using real file names. As it is this could be a problem already if this contained sensitive information.

Please remove this file and replace with an example one that removes the hostnames and uses `.example.com` names you make up.